Makefile

```diff
@@ -52,4 +68,5 @@ cross-arm: ## build static packages
 
 .PHONY: clean
 clean: ## clean the build artifacts
+	$(RM) cri-dockerd
```

Comment:
```suggestion
	rm -f cri-dockerd
```

Reply:
It is the same thing

Reply:
I'm not a fan of letting it be overridden, but fair enough.

Reply:
I'm not a fan of letting it be overridden, but fair enough.